### DISCUSSION TITLE
6.0.0-beta.5

### DISCUSSION THREAD

**Discussion by arturcic** [GitTools/GitVersion#3871] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/GitTools/GitVersion/releases/tag/6.0.0-beta.5'>6.0.0-beta.5</a>.</em>

**Comment by rcdailey** [comment]
Are there docs for v6.0 somewhere? I'm trying it out in my project but I'm not getting the behavior I expect. Right now it seems like `dotnet gitversion` isn't using my `GitVersion.yml`. I changed `mode` but the version output doesn't appear to be any different.

EDIT: There might be a misunderstanding on my part. I was expecting `mode: Mainline` to fail as I believed support for this mode was dropped.

**Reply by rcdailey** [reply]
@arturcic Do you have any comments to share about why beta.5 was not functioning as expected?

**Reply by arturcic** [reply]
I think it's better for @HHobeck  to answer this question

**Reply by HHobeck** [reply]
What do you mean with that? It's a new information for me that beta.5 was not working. Please see the [configuration.md](https://github.com/GitTools/GitVersion/blob/main/docs/input/docs/reference/configuration.md) if you want to know more about the behavior of gitversion and what configuration option you have.

Happy Branching!
